app/src/components/market/common/history_section/history_select.tsx

```diff
@@ -149,7 +149,13 @@ export const History_select: React.FC<Props> = ({
         const response: any[] = await Promise.all(
           fpmmTransactions.map(async item => {
             const safe = new SafeService(item.user.id, context.library)
-            const owners = await safe.getOwners()
+            let owners: string
```

Comment:
This is good, but if there is only one owner then it's slightly less confusing if we change the variable name to `owner`